frontstage/views/sign_in/sign_in.py

```diff
@@ -30,22 +35,17 @@ def login():
     if request.method == 'POST' and form.validate():
         username = request.form.get('username')
         password = request.form.get('password')
-        sign_in_data = {
-            "username": username,
-            "password": password
-        }
-        response = api_call('POST', app.config['SIGN_IN_URL'], json=sign_in_data)
 
-        # Handle OAuth2 authentication errors
-        if response.status_code == 401:
-            error_json = json.loads(response.text).get('error')
-            error_message = error_json.get('data', {}).get('detail')
-            if 'Unauthorized user credentials' in error_message:
+        try:
+            oauth2_token = oauth_controller.sign_in(username, password)
+        except OAuth2Error as exc:
+            error_message = exc.message
+            if BAD_AUTH_ERROR in error_message:
                 return render_template('sign-in/sign-in.html', form=form, data={"error": {"type": "failed"}})
-            elif 'User account locked' in error_message:
+            elif ACCOUNT_LOCKED_ERROR in error_message:
                 logger.debug('User account is locked on the OAuth2 server')
                 return render_template('sign-in/sign-in.trouble.html', form=form, data={"error": {"type": "account locked"}})
```

Comment:
I don't think this is handled by the UI anymore or it shouldn't be we don't want to be feeding back that the account is locked

Reply:
@gemmai95 can this case be removed then? (Should it be done so in this PR or separately?).

Reply:
I have removed it in my PR.

Reply:
I'll drop it out here as well then.